### PR TITLE
PR#8

### PR DESCRIPTION
**Pull request by SivaaB**
This PR addresses Issue #43. 
This will be the last PR I open for this Issue. 

Changes:

1. EG has been updated to include compre. (essentially my repo link with all answers to practice questions on LMS, and I changed the line that said "as of now only mid-sem syllabus") 
2. TRW I added my group's Feedback Report + PPT for reference, we were graded the highest in our batch for both, lots of good takeaway points for juniors on that.

### CODE REVIEW

File: docs/1_Core/Technical_Report_Writing/03_Sample_Report/index.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
+# Sample Report and Slides
 
-[Sample Report Feedback from Turnitin](Report_Feedback_Compressed.pdf)
+- Report - 1 : [The Problems of the Web and How to fix them](Report_Feedback_Compressed.pdf)
+- Report - 2 : [CyberSecurity and Data Leak Detection](https://drive.google.com/file/d/1QrA23Df_pymzKW4TxfoeI4YXoxhEEnT_/view?usp=share_link)
 
-Observe the feedback from the professors, and improve on those points.
+- Slides - 2 : [CyberSecurity and Data Leak Detection - Slides](https://drive.google.com/file/d/1JDh-URd5JPajHgbjY9BUCSKOfaa5_-bx/view?usp=share_link)
+
+Observe the feedback from the professors, and improve on those points. Report - 2 has been provided by a junior, this report and presentation have been graded the highest for IInd Sem, 2022-23 . Thank you Sivaa! ✨
```
Comment (AhmedThahir) [inline docs/1_Core/Technical_Report_Writing/03_Sample_Report/index.md:8]: My man put "thank you sivaa" on his own. Idk what more to say 😭😂

**Comment by AhmedThahir** [comment]
Looks good. Great work for this year Sivaa. Proud of you!

**Comment by AhmedThahir** [comment]
Mahn I'm dumb ... I was doing on my phone and did something. Fix it thanks! 😂

File: docs/1_Core/Technical_Report_Writing/03_Sample_Report/index.md (hunk 1 above)
Comment (SivaaB) [reply docs/1_Core/Technical_Report_Writing/03_Sample_Report/index.md:8]: Classic, I noticed I put my Github Repo with lab codes on it for CSF111 back when I started (where I did not mention myself). You had edited it to that line. So, I copy pasted lol.

**Comment by AhmedThahir** [comment]
I completed the tasks. Enjoy your vacation @SivaaB ! :)

**Pull request merged**